core/data_migration_scripts/EE_DMS_Core_4_1_0.dms.php

```diff
@@ -1553,7 +1553,7 @@ private function _create_image_attachment_from_GUID($guid, EE_Data_Migration_Scr
         $attach_data = wp_generate_attachment_metadata($attach_id, $local_filepath);
         if (! $attach_data) {
             $migration_stage->add_error(sprintf(esc_html__(
-                "Coudl not genereate attachment metadata for attachment post %d with filepath %s and GUID %s. Please check the file was downloaded properly.",
+                "Coudl NOT genereate attachment metadata for attachment post %d with filepath %s and GUID %s. Please check the file was downloaded properly.",
```

Comment:
Leave lowercase so it doesn't break string translations?

Reply:
also